feincms3_sites/models.py

```diff
@@ -28,7 +28,7 @@ def for_host(self, host):
         return default
 
 
-class Site(models.Model):
+class AbstractSite(models.Model):
```

Comment:
django.contrib.auth uses a `base_user` module for its abstract models to avoid importing the `models` file containing the swapped out version if it is not required. Maybe that's not necessary, but it may be nice. What do you think?

Reply:
This would be overly complex for the site model, as the base is already the AbstractSite and everything in that module, i.e. `site` already declares/defines everything that is needed for a basic site.

The user model is rather complex as it deals with security and the site model does not need such complexity as the user model already provides for that. What the site model needs, though, are additional fields that can then be rendered in the templates or used in helper functions to create for example meta data and whatnot.

So I would keep it rather simple and not introduce another level of abstraction.

Reply:
Oh I think I didn't express myself clearly. I didn't want to add any authentication or security, only suggest that it may be easier when separating the abstract models into a `feincms3_sites/base.py` module and keep the concrete model in `feincms3_sites/models.py` or something. 

It probably doesn't matter.

Reply:
Of course not, I just wanted to point out that the requirements for the user model are somewhat more elaborate and that splitting the models of feincms3_sites into separate basemodels and models modules will provide no extra benefit.

Reply:
As long as Django doesn't complain about importing models which aren't migrated or part of `INSTALLED_APPS` that's fine. I'm not sure but I remember that there was trouble in this area once. 

Anyway, if it works it works and that's definitely enough for me .)